### ISSUE TITLE
Missing GraphQL Schema for Setting Column Defaults When Creating a Class

### ISSUE THREAD

**Issue by 185driver** [parse-community/parse-server#6947]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

I don’t see a way to add `required` and `defaultValue` properties to the schema at class creation time.

### Steps to reproduce

Use the `createClass` mutation to create a class with something like:
```js
addNumbers: [
  {
    name: "quantity"
    required: true
    defaultValue: 1
  }
]
```
### Actual Outcome

The last two properties seem to be missing from the schema.

###  Environment

Server
- Parse Server version: `4.3.0`
- Operating system: `Windows 10`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `localhost`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `4.2.8`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `MongoDB Atlas`

Client
- SDK (iOS, Android, JavaScript, PHP, Unity, etc): `GraphQL`
- SDK version: `N/A`

### Comments
Would the ability to set Class Level Permissions via the GraphQL `createClass` mutation be a part of this same issue, or something separate?

**Comment by mtrezza** [comment]
Thank you for reporting.

@Moumouls @davimacedo Since this is GraphQL issue, do you have any suggestions?

**Comment by 185driver** [comment]
Because @davimacedo asked me to create this issue, I believe he plans to care for it.

**Comment by Moumouls** [comment]
@185driver  yeah it seems that my last PR on this case only cover the output type: https://github.com/parse-community/parse-server/pull/6279/files

**Comment by 185driver** [comment]
I appreciate the invite but I'm currently away from my parse server project and building something else so am unable to participate for now. I'll check back when I can.
